client/src/unifyfs.c

```diff
@@ -1333,8 +1361,9 @@ int unifyfs_fid_open(const char* path, int flags, mode_t mode, int* outfid,
         }
 
         if (flags & O_APPEND) {
-            unifyfs_filemeta_t* meta = unifyfs_get_meta_from_fid(fid);
-            pos = meta->size;
+            meta = unifyfs_get_meta_from_fid(fid);
+            LOGDBG("opening append, meta is %p\n", meta);
+            pos = meta->local_size;
```

Comment:
Let's add a comment here on something to the effect that it's ok to use local_size vs global_size, since we assume the global_size only matters after lamination and someone can't write to a laminated file.

Reply:
Will do